editor/filesystem_dock.cpp

```diff
@@ -302,6 +302,8 @@ void FileSystemDock::_notification(int p_what) {
 
 			always_show_folders = bool(EditorSettings::get_singleton()->get("docks/filesystem/always_show_folders"));
 
+			set_file_list_display_mode(FileSystemDock::FILE_LIST_DISPLAY_LIST);
+
 			_update_display_mode();
```

Comment:
I guess this can be removed now, as `set_file_list_display_mode` will already trigger an update.

Reply:
At first sight it seemed to me that `set_file_list_display_mode()` will update the file list display mode (thumbnails or list), while `_update_display_mode() ` will update the display mode (tree only or split).

Experimenting with it, seems that `_update_display_mode()` is actually redundant case.

Reply:
It's further down the call stack, `set_file_list_display_mode()` calls `_toggle_file_display()` which calls `_set_file_display`, which then calls `_update_file_list(true);`.

Reply:
Nevermind, I got things wrong. It doesn't actually call `_update_display_mode()`, so i guess it's safer to keep it. Sorry about that.

Reply:
Yes, it's probably more future proof to leave it there.